content/docs/command-reference/import-url.md

```diff
@@ -124,9 +124,10 @@ source.
   the imported data (`out`).
 
 - `--no-exec` - create `.dvc` file without actually downloading `url`. E.g. if
-  the file or directory already exist it can be used to skip download.
-  `dvc commit <out>.dvc` should be used to calculate the URL and data hash,
-  update the `.dvc` files, and save existing data to the cache.
+  the file or directory already exists, this can be used to skip the download.
+  The data hash is not calculated by this since only the metadata is saved into
+  the `.dvc` file. You can use `dvc commit <out>.dvc` if you need the hashes in
+  the new `.dvc` file (_this also updates the cache_).
```

Comment:
minor - `_this also updates the cache_` - @BurnzZ I think it's better to use `and save existing data to the cache` (for consistency- we don't use the cache update terminology in other places)

Reply:
Updated.